float64 x
float64 y
string orientation
float64 battery
---
float64[] x
float64[] y
string[] orientation
float64[] battery
float64[] g_cost
string[] action